packages/core/utils/src/replaceBundleReferences.js

```diff
@@ -66,7 +66,7 @@ export function replaceURLReferences({
       continue;
     }
 
-    if (!resolved || resolved.bundleBehavior === 'inline') {
```

Comment:
Is this still necessary? This might cause some change even with the current/old bundler

Also isn't this already covered by the  `resolved == null`  if statement just before?